lsp/core/aws-lsp-core/src/content/handlers/fileHandler.ts

```diff
@@ -0,0 +1,22 @@
+import * as fs from 'fs'
+import { promisify } from 'util'
+import { URI } from 'vscode-uri'
+import {
+    ContentRequestMiddleware,
+    ContentRequestMiddlewareDelegate,
+    ContentRequestResponse,
+} from '../contentRequestMiddleware'
+const readFile = promisify(fs.readFile)
+
+export class FileHandler implements ContentRequestMiddleware {
+    async get(uri: URI, next: ContentRequestMiddlewareDelegate): Promise<ContentRequestResponse> {
+        if (!(uri.scheme == 'file')) {
```

Comment:
For my own edification, as it's been awhile and I know JS has some duck typing funkiness, is there a difference between `!(uri.scheme == 'file')` and `uri.scheme != 'file'`?

Reply:
I don't know how that happened, I must have overlooked this during a refactor -- I have changed this to use `!==`